### ISSUE TITLE
Parsedown Extra ?

### ISSUE THREAD

**Issue by qwertygc** [KEINOS/parsedown-extension_table-of-contents#13]
Hello,
[This code](https://github.com/BenjaminHoegh/ParsedownExtended/blob/master/ParsedownExtended.php) use a trick for use Parsedown Vanilla or Parsedown Extra. Your code works with parsedown extra, it's probably  good feature ?

```
if (class_exists('ParsedownExtra')) {
    class DynamicParent extends ParsedownExtra
    {
        public function __construct()
        {
            if (version_compare(parent::version, '0.8.0-beta-1') < 0) {
                throw new Exception('ParsedownExtended requires a later version of ParsedownExtra');
            }
            parent::__construct();
        }
    }
} else {
    class DynamicParent extends Parsedown
    {
        public function __construct()
        {
            if (version_compare(parent::version, '1.8.0-beta-6') < 0) {
                throw new Exception('ParsedownExtended requires a later version of Parsedown');
            }
        }
    }
}


```

regards

**Comment by KEINOS** [comment]
@qwertygc 

I've [released v1.1.0](https://github.com/KEINOS/parsedown-extension_table-of-contents/releases/tag/v1.1.0) featuring your request! Now it's compatible with Parsedown Extra!!

Thanks for the good suggestion!

**Comment by qwertygc** [comment]
thank you :-)
